src/PureScript/CST/Range.purs

```diff
@@ -8,12 +8,13 @@ module PureScript.CST.Range
 import Prelude
 
 import Control.Lazy (defer)
-import Data.Array (foldMap)
 import Data.Array as Array
 import Data.Array.NonEmpty (NonEmptyArray)
 import Data.Array.NonEmpty as NonEmptyArray
+import Data.Foldable (foldMap)
 import Data.Maybe (Maybe(..), maybe)
 import Data.Tuple (Tuple(..), fst, snd)
+import Prim (Array)
```

Comment:
What's this import about?

Reply:
Not sure how that slipped in. Removed.

Reply:
Why do we want to hide everything from Prim except Array. In other places we use `hiding`.

Reply:
Sure, I can use `hiding` instead for consistency and so that this doesn't have to be updated if we use more things from `Prim` in this module.